x/bank/keeper/keeper.go

```diff
@@ -302,6 +304,7 @@ func (k BaseKeeper) SendCoinsFromAccountToModule(
 	return k.SendCoins(ctx, senderAddr, recipientAcc.GetAddress(), amt)
 }
 
+// NOTE(Heimdall-v2): vesting accounts are not used in heimdall
```

Comment:
Same as below, but in general safer to make the logic in this call never gets executed by temporarily returning or something.

Reply:
Done [1cbb41b](https://github.com/0xPolygon/cosmos-sdk/pull/5/commits/1cbb41b4cb73390c58ced14d1e5f76288a1802a3)